src/subcommand/server/accept_encoding.rs

```diff
@@ -0,0 +1,116 @@
+use {super::*, axum::extract::FromRef};
+#[derive(Default)]
+pub(crate) struct AcceptEncoding(pub(crate) String);
```

Comment:
Let's make this an `Option<String>`, also, I think the inner value can be private.